From aaa837cf5ab2db11062f82a7d00d7dae9d7b193d Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 03 Aug 2018 17:24:57 +0300
Subject: [PATCH] dali [linux defconfig]: add CONFIG_MTD_PARTITIONED_MASTER

preperation for backdoor to burn all device
after partition table is changed.
Now one can raw access to device without any partition.
---

diff --git a/board/habanalabs/dali/linux_asic.fragment b/board/habanalabs/dali/linux_asic.fragment
index 0f91d35..d6c797f 100644
--- a/board/habanalabs/dali/linux_asic.fragment
+++ b/board/habanalabs/dali/linux_asic.fragment
@@ -16,3 +16,4 @@
 CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
 CONFIG_SPI_DESIGNWARE=y
 CONFIG_SPI_DW_MMIO=y
+CONFIG_MTD_PARTITIONED_MASTER=y
